app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt

```diff
@@ -55,29 +55,45 @@ fun DebugScreen() {
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun DebugContent(
-    mlsData: List<String>,
-    isLoggingEnabled: Boolean,
+    state: DebugScreenState,
     setLoggingEnabledState: (Boolean) -> Unit,
     logFilePath: () -> String,
     deleteAllLogs: () -> Unit,
     navigateBack: () -> Unit,
     lazyListState: LazyListState = rememberLazyListState()
 ) {
+    val context = LocalContext.current
+    val clipboardManager = LocalClipboardManager.current
     Scaffold(
         topBar = { TopBar(title = "Debug", navigateBack = navigateBack) }
     ) { internalPadding ->
         LazyColumn(
             state = lazyListState,
-            modifier = Modifier.fillMaxSize().padding(internalPadding)
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(internalPadding)
         ) {
             item(key = "mls_data") {
                 ListWithHeader("MLS Data") {
-                    mlsData.map { TextRowItem(it) }
+                    state.mlsData.map { TextRowItem(it) }
                 }
             }
             item(key = "logs") {
                 ListWithHeader("Logs") {
-                    LoggingSection(isLoggingEnabled, setLoggingEnabledState, logFilePath, deleteAllLogs)
+                    LoggingSection(state.isLoggingEnabled, setLoggingEnabledState, logFilePath, deleteAllLogs)
+                }
+            }
+            item(key = "Client ID") {
+                ListWithHeader("Client ID") {
+                    TextRowItem(
+                        state.currentClientId,
+                        trailingIcon = R.drawable.ic_copy
+                    ) {
+                        getDeviceId(context)?.let { AnnotatedString(it) }?.let {
+                            clipboardManager.setText(it)
+                            Toast.makeText(context, "Text Copied to clipboard", Toast.LENGTH_SHORT).show()
```

Comment:
Can we move hardcoded texts to strings file ?

Reply:
it's a debug screen, so this will be use internally , and not sure if we need to localize those

Reply:
as @MohammedMokresh mentioned this screen is for internal use only and we don't need to add any translation for it